.github/labeler.yml

```
@@ -1,20 +1,20 @@
+"Type: CI":
 - changed-files:
   - any-glob-to-any-file: ['.github/*', '**/*.yml', '.gitattributes', '.gitignore']
 
-"PR: JSON":
+"JSON":
 - changed-files:
   - all-globs-to-any-file: ['**/*.json']
 
-"PR: Audio":
+"Audio":
 - changed-files:
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 1

(indentation)